.travis.yml

```diff
@@ -14,10 +14,8 @@ python:
   - "3.5"
   - "3.6"
   - "3.7"
+  - "3.8"
 
-# Workaround for testing Boto3 on Travis. Taken from:
-# https://github.com/azavea/django-amazon-ses/blob/57ed8619435194742308c6f15cbf9c91b231b88a/.travis.yml#L17-L19
-before_install: sudo rm /etc/boto.cfg
```

Comment:
This step was failing the build because `/etc/boto.cfg` no longer exists on the build worker file system.